src/components/CommentCard.jsx

```diff
@@ -12,6 +44,7 @@ const CommentCard = ({ gameComment }) => {
         <p>{gameComment.body}</p>
         <p>Votes: {gameComment.votes}</p>
       </CardContent>
+      <button value = {gameComment.comment_id} onClick={(e) => setDeleteComment(e.target.value)}>Delete Comment</button>
```

Comment:
You don't want to render a delete button on every single comment, only on comments of a particular user that's signed in